core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/ema-contentlet-tools/ema-contentlet-tools.component.ts

```diff
@@ -25,17 +27,20 @@ import { ContentletArea } from '../ema-page-dropzone/types';
     styleUrls: ['./ema-contentlet-tools.component.scss'],
     changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class EmaContentletToolsComponent {
+export class EmaContentletToolsComponent implements OnDestroy {
     private dotMessageService = inject(DotMessageService);
 
     private buttonPosition: 'after' | 'before' = 'after';
 
     @Input() contentlet: ContentletArea;
+    @HostBinding('class.hide') @Input() hide = false;
     @Output() addContent = new EventEmitter<ActionPayload>();
     @Output() addForm = new EventEmitter<ActionPayload>();
     @Output() addWidget = new EventEmitter<ActionPayload>();
     @Output() edit = new EventEmitter<ActionPayload>();
     @Output() delete = new EventEmitter<ActionPayload>();
+    @Output() move = new EventEmitter<ActionPayload>();
+    @Output() dragStop = new EventEmitter<void>();
```

Comment:
I think this is getting out of hand, we can talk about if we should make a refactor to use just one EventEmitter. It could be another ticket tho

Reply:
We will evaluate this in the future if we need to escalate